docs/consume-packages/Finding-and-Choosing-Packages.md

```diff
@@ -147,7 +147,8 @@ For the MSBuild equivalent to setting the license in the `.nuspec`, take a look
 NuGet package search works the same on nuget.org, from the NuGet CLI, and within the NuGet Package Manager extension in
 Visual Studio. In general, search is applied to keywords as well as package descriptions.
 
-- **Filtering**: You can apply a search term to a specific property by using the syntax `<property>:<term>` where
+- **Filtering**: Nuget.org supports search filters. Other package sources depend on server implementation.
```

Comment:
Personally I would move the "other package sources" disclaimer to the bottom

```suggestion
- **Filtering**: nuget.org supports search filters.
```

Reply:
You can add a note at the bottom with something like:

```
> [Note]
> The advanced search syntax is supported by nuget.org. Other package sources - like Azure Artifacts or GitHub Package Registry - may use different search syntaxes. 
```

Reply:
I like the version better as well, but don't feel too strong.y